projects/networking/traditional-web-hosting-on-aws.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Traditional Web Hosting on AWS"
+---
+
+| Challenge name | Cloud(s) | Challenge goal | Contributor |
+| :--- | :--- | :--- | :--- |
+| Traditional Web Hosting on AWS | AWS | Understand how to run your application in a traditional web architecture | [@tonytalkstech](https://twitter.com/tonytalkstech) and [@loujaybee](https://twitter.com/loujaybee) |
+
+### Intro
+
+_Note: I feel like there's an opportunity to discuss IaaS vs. Saas/PaaS here, but I'm not sure if it's a useful conversation. It may help the following paragraphs, as I want to walk away from the "serverless" moniker and replace it with SaaS or Paas. Thoughts are welcome here!_
```

Comment:
I feel like the introduction you've provided is sufficient to get people started - and the initial reading section is super helpful as well.